app/retail/templates/index.html

```diff
@@ -417,6 +417,35 @@ <h4 class="">{% trans "Press" %}</h4>
           </div>
         </div>
       </div>
+      <div class="container-fluid pt-0" style="background-color: #eee;">
+        <div class="row pb-1">
+          <div class="col text-center">
+            <h4 class="">{% trans "iOS App Now Available" %}</h4>
+          </div>
+        </div>
+        <div class="row pb-1">
+          <div class="col d-flex flex-column align-items-center">
+            <div class="explain-img-wrapper mb-3 d-flex justify-content-center align-items-center" style="min-height: 200px;">
+            <a href="{% url "ios" %}">
+                <img style="min-height: 200px;" src="{% static "v2/images/ios/iphone_crop.png" %}" >
```

Comment:
replace it with `max-height: 1rem` ? cause we are shifting to `rem` + that way this doesn't seem to big on mobile devices
we should move this to css file too!